src/app/src/util/constants.js

```diff
@@ -213,6 +213,7 @@ export const LIVE_FEED_MOCK_FISH = {
     timestamp: new Date().getTime(),
     video: LIVE_FEED_URL,
     photo: LIVE_FEED_URL,
+    isLiveFeed: true,
```

Comment:
We detect if the fish is the live feed fish two different ways. I like this field better than a name check (which is what is done here https://github.com/azavea/pwd-fishway/pull/109/files#diff-11b2b3c1e45b70b30d73055c028f1a06R75) and we'll want to propogate that accordingly -- in `HighlightFish.js` add a field `isLiveFeed: bool` and import the `bool` prop-type, but don't make it required.

Update https://github.com/azavea/pwd-fishway/pull/109/files#diff-11b2b3c1e45b70b30d73055c028f1a06R75 to use that field